src/Core/OrganizationFeatures/OrganizationDomains/VerifyOrganizationDomainCommand.cs

```diff
@@ -39,6 +39,13 @@ public async Task<OrganizationDomain> VerifyOrganizationDomain(Guid id)
             throw new ConflictException("Domain has already been verified.");
         }
 
+        var claimedDomain =
+            await _organizationDomainRepository.GetClaimedDomainsByDomainNameAsync(domain.DomainName);
+        if (claimedDomain.Any())
+        {
+            throw new ConflictException("The domain has been verified by another organization.");
```

Comment:
Does this verbiage match the verbiage on save that we return when a domain has already been verified?

Reply:
I changed it to what was reported on Jira. are you okay with the error message from the create endpoint?

Reply:
I think using the same verbiage in both cases would be more consistent. 

On the create endpoint, we used: "The domain is not available to be claimed."  I'm going to condition off it to provide a translated message regardless so it's not that important that they match, but the consistency does add value, I think.

Reply:
yeah that's alright